lib/install/validate/validator.go

```diff
@@ -501,21 +504,147 @@ func (v *Validator) certificateAuthorities(ctx context.Context, input *data.Data
 func (v *Validator) registries(ctx context.Context, input *data.Data, conf *config.VirtualContainerHostConfigSpec) {
 	defer trace.End(trace.Begin(""))
 
+	// Check if CAs can be loaded
+	pool := x509.NewCertPool()
```

Comment:
We should add a means to build the endpoint public registry set into vic-machine so we can use a known set of public certs. That way we won't validate successfully on client machine, but fail on VCH.